identity/cache.go

```diff
@@ -22,22 +22,22 @@ func NewIdentityCache(dir string, jsonFile string) *identityCache {
 	}
 }
 
-func (ic *identityCache) GetIdentity() *dto.Identity {
+func (ic *identityCache) GetIdentity() (identity dto.Identity) {
```

Comment:
`GetIdentity() (dto.Identity, error)` to recognise error about that result is empty